packages/account/src/Sections/Verification/Helpers/verification.ts

```diff
@@ -0,0 +1,45 @@
+import { GetAccountStatus } from '@deriv/api-types';
```

Comment:
Can we just leave this file as it is for now because I'm working on #9402 and this logic can be shared with that hook?

Reply:
yes, sure. 
It’s only TS migration. I'll revert it @shahzaib-deriv